### DISCUSSION TITLE
Signed users - can we have username instead of "Sign In" label ?

### DISCUSSION THREAD

**Discussion by JCAMPOLPA** [Azure/api-management-developer-portal#2548] [Feature request]
Env: Managed

Is there a way to have user name , on the right upper left, instead of the static "Sign In" menu when the user is connected ?
![image](https://user-images.githubusercontent.com/26059633/87937662-6adc9400-ca95-11ea-8776-927d0ff0f7ed.png)

It is perturbing the connected user does not realized he is connected (or may have missed something !!).

**Comment by azaslonov** [comment]
Hi @JCAMPOLPA, I think one of the problems you have here is the Menu visibility config. For instance, in default template, signed-in user looks like this:
![image](https://user-images.githubusercontent.com/2320302/87981985-f6045c80-ca8a-11ea-8c3d-f19637fec68c.png)

Regarding the name of signed-in users, I think that makes sense. We can add this.
